conduit-extra/Data/Conduit/Text.hs

```diff
@@ -61,7 +60,7 @@ import Data.Streaming.Text
 --
 -- Since 0.3.0
 data Codec = Codec
-    { codecName :: T.Text
+    { _codecName :: T.Text
```

Comment:
It's a good documentation strategy to indicate what the purpose of the field is.